### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by Kanyorok**
- Use database transactions.
- Modify and delete data in SQL.
- Prepare complex queries that answer analytical questions.
Transaction_1
Transaction_2
Transaction_3
Transaction_4

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,12 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES ('Gabumon', date '2018-11-15', 2, TRUE, 8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES ('Pikachu', date '2021-01-07', 1, FALSE, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES ('Devimon', date '2017-05-12', 5, TRUE, 11);
+
+/*Day Two Project*/
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', date '2020-02-08', 0, TRUE, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantom', date '2021-11-15', 2, TRUE, 5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', date '1993-04-02', 3, FALSE, 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', date '2005-06-15', 1, TRUE, 45);
```
Comment (Owusu-Desmond) [inline data.sql]: - [ ] As per the project requirement, Please you are inserting the wrong `weight_kg` values for these data below so make sure you fix  them.
  
  image

File: queries.sql [hunk 2]
```diff
@@ -24,6 +24,65 @@ SELECT * FROM animals WHERE name!='Gabumon';
...
+/*How many animals have never tried to escape?*/
+SELECT COUNT(*) from animals WHERE escape_attempts=0;
+/*What is the average weight of animals?*/
+SELECT AVG(weight_kg)::NUMERIC(10,2) As avg_weight from animals;
+/*Who escapes the most, neutered or not neutered animals?*/
+SELECT SUM(escape_attempts) AS escape_attempts,neutered from animals GROUP BY neutered;
+/*What is the minimum and maximum weight of each type of animal?*/
+SELECT MIN(weight_kg) AS minimum_weight,MAX(weight_kg) as max_weight,species from animals GROUP BY species;
+/*What is the average number of escape attempts per animal type of those born between 1990 and 2000?*/
+SELECT MIN(weight_kg) AS min_weight,MAX(weight_kg) as max_weight,species from animals GROUP BY species;
```
Comment (Owusu-Desmond) [inline queries.sql]: - [ ] I suggest that we want the average number of escape attempts per animal type of those born between 1990 and 2000 so I think you add the `WHERE` clause to specify the `date_of_birth` between 1990 and 2000 :blush:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Kanyorok , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

- [ ] From the project requirement, you should Mmodify your `schema.sql` file and add a column `species` of type `string` to your animals table. 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @Kanyorok  , 
# STATUS: APPROVED! 🥇 💯 

done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**